### DISCUSSION TITLE
Unable to login via reverse proxy with 0.3.21

### DISCUSSION THREAD

**Discussion by rcalv002** [open-webui/open-webui#5308] [General]
# Bug Report

## Installation Method

Docker

## Environment

- **Open WebUI Version:** [e.g., v0.3.21

- **Operating System:** Linux
- **Browser (if applicable):** Edge, Brave

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [x] I have included the browser console logs.
- [x] I have included the Docker container logs.
- [x] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

Expected successful login as in previous versions

## Actual Behavior:

As of latest release, unable to login via reverse proxy (been using Nginx Proxy Manager for years) with https://hostname.com, which worked without issue before, and continues to work for my other home lab services. Accessing directly via http://dockerip:exposedport works fine. Have cleared cache, tried different browsers and from different vms

## Description

**Bug Summary:**
Unable to login, i get 502 or 504 errors and socket errors.

## Reproduction Details

**Steps to Reproduce:**
Pull latest release with tag :latest on docker. 
Spin up brand new container with different volume. 
Login via direct http and create okay.
Point nginx proxy manager at this new container.
Open private session on same pc or another vm and try to browse to the secured host name, sits on login screen.

## Logs and Screenshots

**Browser Console Logs:**
![image](https://github.com/user-attachments/assets/e9f5217d-8171-4aaa-9518-03aff018ca63)

**Docker Container Logs:**
![image](https://github.com/user-attachments/assets/b41d16cf-091a-4833-b638-c27d4e4a5dbd)

## Additional Information

Proxy configuration has not changed and maintains websocket support

**Comment by rcalv002** [comment]
I see the issue was closed. any feedback here?

**Reply by justinh-rahb** [reply]
Do you have Websocket support enabled? (Not a specific example, just the first one of this screen of NPM that I could find)

![99294732-5e4a2b80-283c-11eb-8a91-61148eca0990](https://github.com/user-attachments/assets/0ee216b1-19a7-43a8-acf4-4782829baea5)
